package.json

```
@@ -15,12 +15,12 @@
   "devDependencies": {
     "@babel/core": "^7.7.0",
     "@testing-library/user-event": "^13.2.1",
-    "@theforeman/builder": "^8.4.1",
-    "@theforeman/eslint-plugin-foreman": "^8.4.1",
-    "@theforeman/find-foreman": "^8.4.1",
-    "@theforeman/stories": "^8.4.1",
-    "@theforeman/test": "^8.9.0",
-    "@theforeman/vendor-dev": "^8.4.1",
+    "@theforeman/builder": "^10.0",
+    "@theforeman/eslint-plugin-foreman": "^10.0",
+    "@theforeman/find-foreman": "^10.0",
+    "@theforeman/stories": "^10.0",
+    "@theforeman/test": "^10.0",
+    "@theforeman/vendor-dev": "^10.0",
```

On other plugins we started to accept `>= 0` for these versions, to reduce the number of changes for plugins:
```suggestion
    "@theforeman/builder": ">= 0",
    "@theforeman/eslint-plugin-foreman": ">= 0",
    "@theforeman/find-foreman": ">= 0",
    "@theforeman/stories": ">= 0",
    "@theforeman/test": ">= 0",
```
btw, `@theforeman/vendor-dev` isn't needed.

same for the peerDeps:
```json
"peerDependencies": {
  "@theforeman/vendor": ">= 0"
},
```

reference: https://github.com/theforeman/foreman_discovery/blob/develop/package.json#L28

---

I don't have time to finish this up, but this is blocking updates. Could you have a look @Ron-Lavi?

---

Hey Ewoud, I am actually on vacation and can't test it, @mariaaga do you mind taking a look? Thanks

---

Then by all means, enjoy your vacation. No need to reply immediately :)